presets/all-packages/associate-role/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
All these `package.json` files are autogenerated by `preconstruct` and required for it to be able to export independent entry points.